swarm/pss/client/client.go

```diff
@@ -17,6 +18,7 @@ import (
 )
 
 const (
+	rpcTimeout     = time.Second
```

Comment:
[As per the docs](https://golang.org/pkg/context/#pkg-overview), users are discouraged from passing a nil context so we should assume it is always set.